app/components/emoji/emoji.js

```diff
@@ -15,20 +15,23 @@ export default class Emoji extends React.PureComponent {
         customEmojis: PropTypes.object,
         emojiName: PropTypes.string.isRequired,
         literal: PropTypes.string,
+        padding: PropTypes.number,
```

Comment:
Instead of passing this in, couldn't we just wrap this in an extra view when needed? Like
```
<View style={{padding: 5}}>
    <Emoji />
</View>
```
Adding a prop for this seems unnecessary